tokio/src/runtime/thread_pool/worker.rs

```diff
@@ -511,7 +511,7 @@ impl Context {
         core.park = Some(park);
 
         // If there are tasks available to steal, notify a worker
```

Comment:
nit, tioli: maybe worth adding something like
```suggestion
        // If there are tasks available to steal, but this worker is not
        // looking for tasks to steal, notify another worker.
```